### ISSUE TITLE
bad pointer from perlfunc to perlmod

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#3334]
Migrated from [rt.perl.org#5637](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=5637) (status was 'resolved')

Searchable as RT5637$

**Comment by p5pRT** [comment]
### From abela@geneanet.org




---
perlfunc says&#8203;:

&nbsp;               Currently implemented pragmas are&#8203;:

&nbsp;                   use integer;
&nbsp;                   use diagnostics;
&nbsp;                   use sigtrap  qw\(SEGV BUS\);
&nbsp;                   use strict   qw\(subs vars refs\);
&nbsp;                   use subs     qw\(afunc blurfl\);
&nbsp;                   use warnings qw\(all\);

&nbsp;               \[\.\.\.\]

&nbsp;               See the perlmod manpage for a list of standard
&nbsp;               modules and pragmas\.

But the perlmod doesn't talk about pragmas \(they are just mentionned\,
there is no such list of standard pragmas\)\.


Jerome\.



---


---
Flags&#8203;:
&nbsp;    category=docs
&nbsp;    severity=medium


---
Site configuration information for perl v5\.6\.0&#8203;:

Configured by pixel at Fri Mar 31 16&#8203;:32&#8203;:49 CEST 2000\.

Summary of my perl5 \(revision 5\.0 version 6 subversion 0\) configuration&#8203;:
&nbsp;  Platform&#8203;:
&nbsp;    osname=linux\, osvers=2\.2\.15\-0\.16mdksmp\, archname=i386\-linux
&nbsp;    uname='linux kenobi\.mandrakesoft\.com 2\.2\.15\-0\.16mdksmp \#1 smp mon mar 13 16&#8203;:40&#8203;:10 cet 2000 i686 unknown '
&nbsp;    config\_args='\-des \-Dprefix=/usr \-Darchname=i386\-linux \-Dd\_dosuid \-Dd\_semctl\_semun \-Di\_db \-Di\_ndbm \-Di\_gdbm \-Ud\_csh \-Dman3dir=/usr/lib/perl5/man/man3 \-Doptimize=\-O3 \-fomit\-frame\-pointer \-fno\-exceptions \-fno\-rtti \-pipe \-s \-mpentium \-mcpu=pentium \-march=pentium \-ffast\-math \-fexpensive\-optimizations'
&nbsp;    hint=recommended\, useposix=true\, d\_sigaction=define
&nbsp;    usethreads=undef use5005threads=undef useithreads=undef usemultiplicity=undef
&nbsp;    useperlio=undef d\_sfio=undef uselargefiles=define 
&nbsp;    use64bitint=undef use64bitall=undef uselongdouble=undef usesocks=undef
&nbsp;  Compiler&#8203;:
&nbsp;    cc='cc'\, optimize='\-O3 \-fomit\-frame\-pointer \-fno\-exceptions \-fno\-rtti \-pipe \-s \-mpentium \-mcpu=pentium \-march=pentium \-ffast\-math \-fexpensive\-optimizations'\, gccversion=2\.95\.3 19991030 \(prerelease\)
&nbsp;    cppflags='\-fno\-strict\-aliasing'
&nbsp;    ccflags ='\-fno\-strict\-aliasing \-D\_LARGEFILE\_SOURCE \-D\_FILE\_OFFSET\_BITS=64'
&nbsp;    stdchar='char'\, d\_stdstdio=define\, usevfork=false
&nbsp;    intsize=4\, longsize=4\, ptrsize=4\, doublesize=8
&nbsp;    d\_longlong=define\, longlongsize=8\, d\_longdbl=define\, longdblsize=12
&nbsp;    ivtype='long'\, ivsize=4\, nvtype='double'\, nvsize=8\, Off\_t='off\_t'\, lseeksize=8
&nbsp;    alignbytes=4\, usemymalloc=n\, prototype=define
&nbsp;  Linker and Libraries&#8203;:
&nbsp;    ld='cc'\, ldflags =' \-L/usr/local/lib'
&nbsp;    libpth=/usr/local/lib /lib /usr/lib
&nbsp;    libs=\-lnsl \-ldl \-lm \-lc \-lposix \-lcrypt
&nbsp;    libc=/lib/libc\-2\.1\.3\.so\, so=so\, useshrplib=false\, libperl=libperl\.a
&nbsp;  Dynamic Linking&#8203;:
&nbsp;    dlsrc=dl\_dlopen\.xs\, dlext=so\, d\_dlsymun=undef\, ccdlflags='\-rdynamic'
&nbsp;    cccdlflags='\-fpic'\, lddlflags='\-shared \-L/usr/local/lib'

Locally applied patches&#8203;:
&nbsp;    



---
@&#8203;INC for perl v5\.6\.0&#8203;:
&nbsp;    /usr/lib/perl5/5\.6\.0/i386\-linux
&nbsp;    /usr/lib/perl5/5\.6\.0
&nbsp;    /usr/lib/perl5/site\_perl/5\.6\.0/i386\-linux
&nbsp;    /usr/lib/perl5/site\_perl/5\.6\.0
&nbsp;    /usr/lib/perl5/site\_perl/5\.005/i386\-linux
&nbsp;    /usr/lib/perl5/site\_perl/5\.005
&nbsp;    /usr/lib/perl5/site\_perl
&nbsp;    \.



---
Environment for perl v5\.6\.0&#8203;:
&nbsp;    HOME=/home/abela
&nbsp;    LANG=fr
&nbsp;    LANGUAGE=fr
&nbsp;    LC\_ALL=fr\_FR
&nbsp;    LD\_LIBRARY\_PATH \(unset\)
&nbsp;    LOGDIR \(unset\)
&nbsp;    PATH=/usr/X11R6/bin&#8203;:/usr/local/bin&#8203;:/bin&#8203;:/usr/bin&#8203;:/home/abela/bin&#8203;:/usr/java/jdk1\.3/bin
&nbsp;    PERL\_BADLANG \(unset\)
&nbsp;    SHELL=/bin/bash

**Comment by p5pRT** [comment]
### From [Unknown Contact. See original ticket]

Lightning flashed\, thunder crashed and abela@&#8203;geneanet\.org whispered&#8203;:
| perlfunc says&#8203;:
| 
|                See the perlmod manpage for a list of standard
|                modules and pragmas\.

<details><summary>Inline Patch</summary>

```diff
--- perlfunc.orig       Thu Feb  1 09:27:19 2001
+++ perlfunc.pod        Thu Feb  1 09:27:55 2001
@@ -5524,7 +5524,7 @@
 
 If no C<unimport> method can be found the call fails with a fatal error.
 
-See L<perlmod> for a list of standard modules and pragmas.  See L<perlrun>
+See L<perlmodlib> for a list of standard modules and pragmas.  See L<perlrun>
 for the C<-M> and C<-m> command-line options to perl that give C<use>
 functionality from the command-line.
```
</details>
